Makefile

```diff
@@ -325,6 +326,36 @@ release-qa: require-tag require-gh-token require-s3cmd require-ruby
 		echo "Copying $$VERSIONED" && \
 		$(S3CMD) cp s3://$(S3_BUCKET)/$$NAME s3://$(S3_BUCKET)/$$VERSIONED; \
 	done && \
+	for NAME in $$(ls -1 update_darwin_amd64.bz2 update_linux_386.bz2 update_linux_amd64.bz2 update_windows_386.bz2); do \
```

Comment:
We are going to need those binaries later and it's possible to lose them if we continue working, so we better save them for later.